functions/src/handlers/ApproveClassHandler.ts

```diff
@@ -135,14 +142,15 @@ export function approveclass (req, res) {
             if (userType === 'teacher') {
                 addToPendingList(req.body);
             } else {
-                return res.status(400).send({message: 'User can\'t approve emails (not teacher).'})
+                return res.status(403).send({message: 'User can\'t approve emails (not teacher).'})
```

Comment:
Can you explain that how 400 and 403 is going to effect the API?

Reply:
So, it doesn't affect the API, it more so helps the client understand what happened. This helps us conform with the [IETF standards](https://www.ietf.org/assignments/http-status-codes/http-status-codes.xml)